QtToastify/qml/Toastify/Toastify.qml

```diff
@@ -16,6 +16,15 @@ Item {
         spacing: 0
     }
 
+    //Top-Center
+    ColumnLayout{
+        id: topCenterColumn
+        x: width / 2
+        y: toprow.y + toprow.height + 1 * g_item_spacing
```

Comment:
Ugh, I just noticed `g_item_spacing`. That's a global from my own project and needs to be removed.